src/types/node.ts

```diff
@@ -34,7 +35,7 @@ export interface CustomNodeConfig extends NodeBase {
 
 export interface StaticNodeConfig extends NodeBase {
   isCustom?: false;
-  type: NodeType.ETHERSCAN | NodeType.INFURA | NodeType.RPC | NodeType.WEB3;
+  type: NodeType.ETHERSCAN | NodeType.INFURA | NodeType.POCKET | NodeType.RPC | NodeType.WEB3;
```

Comment:
Since you added a new `NodeType`, can you just check https://github.com/MyCryptoHQ/MyCrypto/blob/master/src/services/EthService/network/helpers.ts#L15-L42 and verify that Ethers.js should be treating your node as a default `JsonRpcProvider`?

Reply:
Checked; treating Pocket Gateway as a JsonRpcProvider is appropriate!

Reply:
Thanks!